generic_config_updater/field_operation_validators.py

```diff
@@ -20,7 +51,48 @@ def rdma_config_update_validator():
     if branch_version is None:
         return False
 
-    if asic_type == 'cisco-8000':
-        return branch_version >= "20201200"
-    else:
-        return branch_version >= "20181100"
+    # PFCWD enable/disable scenario
+    if "pfc_wd" in field:
```

Comment:
suggest to build a dictionary of the fields and pre-populate it with all the restrictions and write a generic function to evaluate it.
something similar to the eg. below
field = {'pfcwd': {'all': '201811',
                         'cisco-8000': '202012'
                         }}
                         '

Reply:
Thanks for the suggestion, I added this to the gcu_field_operation_validators.conf.json file